test/utils/test_helpers.h

```diff
@@ -90,6 +90,7 @@ class TestData {
     TestData& operator=(const TestData& other)
     {
         data_ = other.data_;
+        return (*this);
```

Comment:
Could we drop the parentheses? I thing they're not needed in this case.

Reply:
fixed